docs/source/user/faq.rst

```diff
@@ -272,6 +272,12 @@ value, for example::
 Miscellaneous
 =============
 
+Where does the project name Numba come from?
+--------------------------------------------
```

Comment:
This line of `'-'` will need extending WRT the above.

Reply:
So I should accept your suggestions, then do a follow-up commit, creating 3 commits in total for this PR?

Reply:
Up to you how you choose to do it. I, personally, would opt for just creating a single patch that makes the suggested changes and github will pick up on them.